### ISSUE TITLE
Crash when selection "Empty space" option in Preferences>colours

### ISSUE THREAD

**Issue by FatChucky** [GeorgRottensteiner/C64Studio#37]
I have found that if you click on the "Empty space" option after selecting multiple custom colours in other options, C64Studio will crash to desktop.

This does seem to be consistent and repeatable.

**Comment by GeorgRottensteiner** [comment]
I tried everything for about 5 minutes but couldn't reproduce. Could you maybe explain more in detail what exactly you're doing? 

Also, which version of C64Studio is this? I fixed a Stack Overflow issue after the release of 6.7 (when Empty Space background was set to "Auto" which sent it in an infinite loop).

Could you try with the preview version from https://www.georg-rottensteiner.de/webmisc/C64StudioRelease.zip ?

**Comment by FatChucky** [comment]
Thanks for the swift reply!

I'm using V.6.7 and trying to set up a dark background. The theme I've got to so far is attached. What I've been doing is stepping down through all the "Element Foreground" colours to dull them down a little as the contrast is too high with the default intense colours.

I should say that I'm using the stand-alone application and not the installed version if that helps. I've tested it again today and can still reproduce it. I tried to record what I was doing but couldn't seem to capture the preferences window. I'll try again later once after dinner if you think seeing this crash happen would help.

I hope that helps.

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by GeorgRottensteiner** [comment]
Anything helping me reproduce this crash is a good thing!

**Comment by FatChucky** [comment]
I installed OSB and recorded the crash:
https://drive.google.com/file/d/1o8PK3gVf7DKGrv4wv22FAvh6PAE5uxIW/view?usp=sharing

The default recording quality was surprisingly poor, but I think it gets the point across.

Let me know if you want a better quality version and I'll look at OSB's settings.

Thanks for your attention!

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by FatChucky** [comment]
Just a wee update. I downloaded your preview version and I wasn't able to reproduce the bug any more. I've no idea what changed but I'm liking being able to tweak my theme.

I just wanted to thank you for creating this IDE. I'm just learning how to code 6502 assembly after almost 20 years away from any kind of coding and am finding it a great way to work. It certainly is quicker to get into than using text editors and a terminal like I used to.

Kudos to you sir!

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
